src/components/controllers/repl-page.jsx

```diff
@@ -0,0 +1,14 @@
+import { Repl } from './repl';
+import { useContent } from '../../lib/use-resource';
+import { useTitle, useDescription } from './utils';
+import { useLanguage } from '../../lib/i18n';
+
+export default function ReplPage() {
```

Comment:
It was the only page left out. This fixes the i18n title as it was hard-coded before, and can allow for a bit of refactoring in the future perhaps.